vm/src/class.rs

```diff
@@ -71,6 +72,7 @@ where
     const TP_NAME: &'static str = T::TP_NAME;
     const DOC: Option<&'static str> = T::DOC;
     const BASICSIZE: usize = T::BASICSIZE;
+    const UNHASHABLE: bool = false;
```

Comment:
`PyRef<T>` must follows `T`

```suggestion
    const UNHASHABLE: bool = T::UNHASHABLE;
```